packages/core/backend/sirius-components-collaborative/src/main/java/org/eclipse/sirius/components/collaborative/editingcontext/EditingContextEventProcessor.java

```diff
@@ -326,6 +330,15 @@ private void handleRepresentationInput(One<IPayload> payloadSink, IRepresentatio
         } else {
             this.logger.warn("No representation event processor found for event: {}", representationInput);
         }
+
+        var processors = this.representationEventProcessors.values().stream().map(RepresentationEventProcessorEntry::getRepresentationEventProcessor).toList();
+
+        // @formatter:off
+        this.editingContextEventHandlers.stream()
+                .filter(handler -> handler.canHandle(this.editingContext, representationInput))
+                .findFirst()
+                .ifPresent(handler -> handler.handle(payloadSink, this.changeDescriptionSink, this.editingContext, representationInput, processors));
+        // @formatter:on
```

Comment:
Since the editing context holds the representation descriptions, it needs to be able to answer the question / input: "what is the representation description of the representation with the given id?"